app/overrides/add_activation_keys_input.rb

```diff
@@ -15,7 +15,7 @@
 
 Deface::Override.new(:virtual_path => "hosts/_form",
                      :name => "hosts_update_environments_select",
-                     :insert_before => 'erb[loud]:contains("host_puppet_environment_field")',
+                     :insert_after => 'erb[loud]:contains("Deploy on")',
```

Comment:
Yes, this is using the label - deface looks at the source files, so as long as https://github.com/theforeman/foreman/blob/develop/app/views/hosts/_form.html.erb#L54 contains 'Deploy on', it'll work, no need to worry about i18n here 😄